src/main/java/org/csanchez/jenkins/plugins/kubernetes/NoDelayProvisionerStrategy.java

```diff
@@ -0,0 +1,64 @@
+package org.csanchez.jenkins.plugins.kubernetes;
+import hudson.Extension;
+import hudson.model.Label;
+import hudson.model.LoadStatistics;
+import hudson.slaves.Cloud;
+import hudson.slaves.NodeProvisioner;
+import jenkins.model.Jenkins;
+
+import java.util.Collection;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+
+/**
+ * Implementation of {@link NodeProvisioner.Strategy} which will provision a new node immediately as
+ * a task enter the queue.
+ * Now that Kubernetes is billed by the minute, we don't really need to wait before provisioning a new node.
```

Comment:
This sentence should be removed as it does not really apply here (comes from EC2 plugin)

Reply:
fix